packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -228,8 +233,8 @@ class DateTimeUtils {
    * @param {boolean} hasSeconds - If true seconds will be converted
    * @return {object} - The moment object representing the given date and time.
    */
-  static convertDateTimeStringToMomentObject(date, time, dateformat, hasSeconds) {
-    return DateTimeUtils.updateTime(DateTimeUtils.createSafeDate(DateUtil.convertToISO8601(date, dateformat)), time, hasSeconds);
+  static convertDateTimeStringToMomentObject(date, time, dateformat, hasSeconds, timeZone) {
+    return DateTimeUtils.updateTime(DateTimeUtils.createSafeDate(DateUtil.convertToISO8601(date, dateformat), timeZone), time, hasSeconds);
```

Comment:
not going to lie, this is a very lengthy return. i wouldn't mind splitting these function calls into separate variables to reduce this.

Reply:
updated here 1e47747132f83358d17d2aaed1f98c6bd4a5cfaf